Documentation/Updating.md

```diff
@@ -1,13 +1,13 @@
-# Updating to the General Availability (GA) Release
+# Updating to the 2.0.0 Release
 
-Between the RC2 and GA releases of the Microsoft Mixed Reality Toolkit, changes were made that may impact existing projects. This document describes those changes and how to update projects to the GA release.
+Between the RC2 and 2.0.0 releases of the Microsoft Mixed Reality Toolkit, changes were made that may impact existing projects. This document describes those changes and how to update projects to the 2.0.0 release.
 
 - [API changes](#api-changes)
 - [Assembly name changes](#assembly-name-changes)
 
 ## API changes
 
-Since the release of RC2, there have been a number of API changes including some that may break existing projects. The following sections describe the changes that have occurred between the RC2 and GA releases.
+Since the release of RC2, there have been a number of API changes including some that may break existing projects. The following sections describe the changes that have occurred between the RC2 and 2.0.0 releases.
```

Comment:
At line 24, can you fix the broken link?
To access extension services, use [`MixedRealityServiceRegistry.TryGetService<T>`].

Reply:
done